### PR TITLE
Chore/updates latest ocr2vrf contract changes

### PR DESCRIPTION
**Pull request by jinhoonbang**

### CODE REVIEW

File: core/scripts/ocr2vrf/util.go [hunk 1]
```diff
@@ -342,7 +342,7 @@ func setBeaconBilling(e helpers.Environment, vrfBeaconAddr string, maximumGasPri
 }
 
 func setCoordinatorBilling(e helpers.Environment, vrfCoordinatorAddr string, useReasonableGasPrice bool, unusedGasPenaltyPercent uint8,
-	stalenessSeconds, redeemableRequestGasOverhead, callbackRequestGasOverhead, premiumPercentage, reasonableGasPriceStalenessBlocks uint32,
+	stalenessSeconds, redeemableRequestGasOverhead, callbackRequestGasOverhead, reasonableGasPriceStalenessBlocks uint32, premiumPercentage uint8,
 	fallbackWeiPerUnitLink *big.Int) {
 	coordinator := newVRFCoordinator(common.HexToAddress(vrfCoordinatorAddr), e.Ec)
```
Comment (makramkd) [inline core/scripts/ocr2vrf/util.go:347]: Can't comment on that line but there's an unnecessary cast `uint8(premiumPercentage)` below.

File: core/scripts/vrfv2/testnet/util.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ func eoaFundSubscription(e helpers.Environment, coordinator vrf_coordinator_v2.V
 	fmt.Println("Initial account balance:", bal, e.Owner.From.String(), "Funding amount:", amount.String())
 	b, err := utils.ABIEncode(`[{"type":"uint64"}]`, subID)
 	helpers.PanicErr(err)
-	e.Owner.GasLimit = 500000
+	e.Owner.GasLimit = 500000000
```
Comment (makramkd) [inline core/scripts/vrfv2/testnet/util.go:89]: I think we should remove this line, without it works just fine on arbitrum.

Comment (makramkd) [reply core/scripts/vrfv2/testnet/util.go:89]:
```suggestion
```

File: core/scripts/ocr2vrf/util.go (hunk 1 above)
Comment (jinhoonbang) [reply core/scripts/ocr2vrf/util.go:347]: oh right. fixed it

**Review by makramkd — APPROVED** [review]

**Pull request closed without merging**